azurerm/internal/services/kusto/tests/resource_arm_kusto_cluster_test.go

```diff
@@ -176,6 +205,28 @@ resource "azurerm_kusto_cluster" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomString)
 }
 
+func testAccAzureRMKustoCluster_propsUpdate(data acceptance.TestData) string {
```

Comment:
Could we make this a complete test, so we can then go basic -> complete -> basic in the update test
```suggestion
func testAccAzureRMKustoCluster_update(data acceptance.TestData) string {
```